pkg/types/validation/installconfig.go

```diff
@@ -699,3 +703,35 @@ func validateFIPSconfig(c *types.InstallConfig) field.ErrorList {
 	}
 	return allErrs
 }
+
+// validateCapabilities checks if additional, optional openshift components are specified in the
+// install-config to be included in the installation
+func validateCapabilities(c *types.Capabilities, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if c.BaselineCapabilitySet == "" {
+		allErrs = append(allErrs, field.Required(fldPath.Child("baselineCapabilitySet"), "baseline capability set required"))
```

Comment:
This is [optional in the ClusterVersion API][1], so I dunno if we need this guard.

[1]: https://github.com/openshift/api/blob/b632c5fc10c0ea86cb18577db9388109a43d465f/config/v1/types_cluster_version.go#L281-L282

Reply:
Folding into [this thread][1], which is making the same point.

[1]: https://github.com/openshift/installer/pull/5629#discussion_r804093636